RefRed/configuration/loading_configuration.py

```diff
@@ -85,22 +86,24 @@ def check_config_file(self, filename):
         except:
             # If we can't parse the file, it's not a valid file
             return False
-        version_tag = dom.getElementsByTagName('version')
+        version_tag = dom.getElementsByTagName("version")
         if len(version_tag) == 0:
-            warning_msg = "The reduction parameters you are about to load are from "
-            warning_msg += "an older version of the reduction.\n\n"
-            warning_msg += "It is NOT advised to mix reduction versions.\n\n"
-            warning_msg += "You should either start refred version 4, or "
-            warning_msg += "reprocess all your data with the new version.\n\n"
-            warning_msg += "Use the following to start version 4:\n\n"
-            warning_msg += "    refred --v4\n\n"
-            warning_msg += "Please consult with your local contact for advise."
+            warning_msg = (
+                "The reduction parameters you are about to load are from "
+                + "an older version of the reduction.\n\n"
+                + "It is NOT advised to mix reduction versions.\n\n"
+                + "If you need to change your reduction parameters,"
+                + "it is recommended to reprocess all your data with the new version.\n\n"
+                + "Use the following to start version 4:\n\n"
```

Comment:
Since we're going to remove `refred --v4`,  we should not ask User to use it.

Reply:
True. removed that line